### ISSUE TITLE
Change NN architecture for small datasets

### ISSUE THREAD

**Issue by jacopok** [jacopok/mlgw_bns#33]
Several techniques could be useful: 

- MC dropout
- have an initial, fixed convolutional layer (or PCA?)
- extra inputs bypassing the pre-trained layer?
- data augmentation? virtual adversarial training
- raise the learning rate for the NR data?
